core/sdk/src/api/firebolt/fb_capabilities.rs

```diff
@@ -159,6 +160,46 @@ pub struct FireboltPermission {
     pub role: CapabilityRole,
 }
 
+impl FireboltPermission {
+    pub fn from_vec_string(
+        perm_strings: Vec<String>,
+        role_based_support: bool,
+    ) -> Vec<FireboltPermission> {
+        let mut perm_list: Vec<FireboltPermission> = Vec::new();
+        for perm in perm_strings {
+            if role_based_support {
+                perm_list.push(FireboltPermission::deserialize(json!(perm)).unwrap());
```

Comment:
please fix the unsafe unwraps we can ignore Firebolt permissions which are not deserializable